entente/landmarks/landmarker.py

```diff
@@ -81,9 +81,10 @@ def transfer_landmarks_onto(self, target):
         if not have_same_topology(self.source_mesh, target):
             raise ValueError("Target mesh must have the same topology")
 
-        return dict(
-            zip(
-                self.landmarks.keys(),
-                apply_surface_regressor(self._regressor, target.v),
-            )
-        )
+        return [
+            {
+                "name": point["name"],
+                "point": apply_surface_regressor(self._regressor, target.v)[i].tolist(),
+            }
+            for (i, point) in enumerate(self.landmarks)
+        ]
```

Comment:
This seems not great.  But I couldn't figure out how to zip it into a list of dicts.

Reply:
Listcomp is the right way to go, but restore the `zip()` as it was and iterate over that. It gives you tuples `(name, point)`.